airtime_mvc/application/services/PodcastEpisodeService.php

```diff
@@ -416,7 +424,8 @@ public function _getImportedPodcastEpisodeArray($podcast, $episodes) {
                 "description" => htmlspecialchars($item->get_description()),
                 "pub_date" => $item->get_gmdate(),
                 "link" => $item->get_link(),
-                "enclosure" => $item->get_enclosure(),
+                // $enclosure now holds only an audio enclosure. JBC
+                "enclosure" => $enclosure,
```

Comment:
`$enclosure` might need initialising or a guard to ensure we never pass it as `null`.